packages/admin-scripts/src/infrastructure/script-executor-handler.js

```diff
@@ -0,0 +1,75 @@
+const { createScriptRunner } = require('../application/script-runner');
+const { createAdminScriptCommands } = require('@friggframework/core/application/commands/admin-script-commands');
+
+/**
+ * SQS Queue Worker Lambda Handler
+ *
+ * Processes script execution messages from AdminScriptQueue
+ */
+async function handler(event) {
+    const results = [];
+
+    for (const record of event.Records) {
+        const message = JSON.parse(record.body);
+        const { scriptName, executionId, trigger, params } = message;
+
+        console.log(`Processing script: ${scriptName}, executionId: ${executionId}`);
+
+        try {
+            const runner = createScriptRunner();
+            const commands = createAdminScriptCommands();
+
+            // If executionId provided (async from API), update existing record
+            if (executionId) {
+                await commands.updateScriptExecutionStatus(executionId, 'RUNNING');
+            }
```

Comment:
Feels to me that the logic inside of the _try_ should be a use case like `handle-Queued-script-use-case` maybe.